app/assets/javascripts/discourse/app/widgets/topic-timeline.js

```diff
@@ -351,6 +351,22 @@ createWidget("timeline-footer-controls", {
     const controls = [];
     const { currentUser, fullScreen, topic, notificationLevel } = attrs;
 
+    if (
+      this.siteSettings.summary_timeline_button &&
+      !fullScreen &&
+      topic.has_summary &&
+      !topic.postStream.summary
+    ) {
+      controls.push(
+        this.attach("button", {
+          className: "show-summary btn-small",
```

Comment:
We should put an icon here. I feel like our buttons look weird when we don't have one! Does the summarize button in the topic map have one?

Reply:
No, we don't use an icon there. It's a little tricky to find an appropriate one. 

Maybe https://fontawesome.com/v5.15/icons/layer-group?style=solid or https://fontawesome.com/v5.15/icons/stream?style=solid?

Reply:
Here's what this look like with the layer-group icon added to both buttons: 

<img width="1158" alt="image" src="https://user-images.githubusercontent.com/368961/133657862-23693639-18ec-41df-a19e-acfd1f808035.png">

Reply:
That's a really nice icon!